fundamentals/minimal-apis/samples/IResultImplementation/ContactsHandler.cs

```diff
@@ -0,0 +1,100 @@
+using IResultImplementation.Data;
+using IResultImplementation.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace IResultImplementation
+{
+    public static class ContactsHandler
+    {
+
+        // GET: api/Contacts
+        public static async Task<IResult> GetContacts(IResultImplementationContext context)
```

Comment:
When using `TypesResults` static methods is recommended to return the IResult implementation type, in this case `Ok<List<Contact>>`

Reply:
https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/responses?view=aspnetcore-7.0#typedresults-vs-results

Reply:
I did that because of it seemed difficult to write test for that when using minimal APIs

Reply:
Ah! You want to demonstrate who you can use the public types in a unit test for a IResult return delegate, right?

Reply:
Yep